applications/TreasuryTracker.md

```diff
@@ -0,0 +1,180 @@
+
+### Team Code Repos
+
+Company:
+- https://github.com/stake-plus
+
+Team:
+- https://github.com/senseless
+- https://github.com/Bluespottedfrog
+
+Reference: https://github.com/orgs/ibp-network/repositories
+
+### Team LinkedIn Profiles (if available)
+
+We no longer use LinkedIn but can provide CV privately if required.
+
+## Development Status :open_book:
+
+At the current stage of the project, extensive research and preparatory work have been carried out to lay the foundation for successful grant application. Below is a detailed description of the work conducted before applying for the grant:
+
+UI Mockups and Functionality:
+Prior to the proposal, careful planning and design were undertaken to create UI mockups that accurately represent the intended functionality of the platform. These mockups serve as a guide for the development process, ensuring that user experience and visual aesthetics align with the project's objectives and future plans.
+
+On-Chain Data Reading:
+Initial testing and development of functions for reading on-chain data have been performed. This represents a crucial step in achieving the project's goals, as it allows for accurate and real-time retrieval of referenda and treasury data directly from the blockchain.
+
+Research and Community outreach:
+Comprehensive research has been conducted to understand the needs of the community, the gaps in existing solutions, and the potential challenges in implementing the project.
+
+
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:** 1 month
+- **Full-Time Equivalent (FTE):**  1.25
+- **Total Costs:** 10,000 USD
+
+### Milestone 1 Example � Basic functionality 
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can use the application and its various features. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article** that explains what was done/achieved as part of the grant |
+| 1. | Backend Development | We will create a backend module that can accurately and efficiently read data from the blockchain  to extract relevant data and store it in a PostgreSQL database. The focus of this phase is to establish a robust data pipeline that ensures the integrity and reliability of the data used for analytics.|
```

Comment:
Here you're referring to PostgreSQL while earlier you mentioned MySQL. Could you clarify?

Reply:
There were multiple people working on the application and I missed it during my review of the milestone sections. We will be using MySQL.